app/controllers/integrations_controller.rb

```diff
@@ -0,0 +1,20 @@
+require 'yaml'
+
+class IntegrationsController < ApplicationController
+  def show
+    begin
+      render json: Integration.find(params[:id])
+    rescue Integration::NotFound => e
```

Comment:
I think this error is a little bit different than the others. This one is just a public facing 404. The code here never even touches the Integrations module either so I think it might be confusing to raise an error from there.